db/migrate/20221126182411_create_comment.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+      t.timestamps
+    end
+  end
+end
```

Comment:
[OPTIONAL] You could have added relationships to it while creating the tables instead of separately:
```ruby
 create_table :comments do |t|
    t.text :text
    t.timestamps # this will crate uopdated_at and created_at columns
    t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
    t.references :post, foreign_key: true # this will create post_id and a corresponding index (but students can also add column and index in separate commands)
  end
```